Lighthouse Camera/UI/Main/PreviewView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  PreviewView.swift
+//  Lighthouse Camera
+//
+//  Created by Punam Dahiya on 1/27/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+protocol PreviewViewDelegate: class {
+  func displayPreviewSession()
+}
+
+// Displays what the camera lens sees in UIView, previewView to
+// see a live video stream of phone camera's input within previewView
+
+class PreviewView: UIView {
+  // This delegate refers to a reference type and that class will be weakly retained
```

Comment:
Nit: I don't think that this comment adds anything to the line below. Could you rather explain *why* it's a `weak`?